draft-ietf-rats-corim.md

```diff
@@ -1194,7 +1195,22 @@ applies to all measurements in the triple, including those in `measurement-value
 
 ## Extensibility {#sec-extensibility}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/10
+CoRIM is described using CDDL {{-cddl}} which includes extensibility support for "sockets" (a.k.a., map structures)
+and data types. CDDL socket extensions have the form `($$NAME-extension)` where "NAME" is the name of the map.
+CDDL socket extensibility enables a CoRIM profile to extend the standard definition.
+Typically, map extension requires a convention for code point naming that avoids code-point resue.
+Well-known code points may be in a registry, such as CoSWID {{-coswid-reg}}.
+Additionally, a range of code points may be reserved for vendor-specific use such as negative integers.
+
+Data type extensibility has the form `($NAME-type-choice)` where "NAME" is the type name and '$' signifies type extensibility.
```

Comment:
Ah. Here we go. I'd move the referrals and definitions to type- and group-choice to the top. And come back to it here.